src/SubscribeMe/Subscriber/MailchimpSubscriber.php

```diff
@@ -113,13 +119,82 @@ public function subscribe(string $email, array $options, array $userConsents = [
                      */
                     return true;
                 }
+                if ($body['id'] !== null) {
+                    return $body['id'];
+                }
+                return false;
             }
-
-            throw new CannotSubscribeException($exception->getMessage(), $exception);
-        } catch (RequestException $exception) {
+        } catch (ClientExceptionInterface $exception) {
             throw new CannotSubscribeException($exception->getMessage(), $exception);
         }
 
         return false;
     }
+
+    /**
+     * @see https://mailchimp.com/developer/transactional/api/messages/send-using-message-template/
+     * @param array<EmailAddress> $emails
```

Comment:
You do not need to repeat PHPDocs for all implementation here. Just use `@inheritDoc`  instead of all `@param` directives